Utilities/Maintenance/ApplyScriptToRemotes.sh

```diff
@@ -122,7 +121,7 @@ function list_candidate_remotes() {
 
     # Get the latest git commit hash of the remote module.
     # Remotes will usually not be tagged.
-    latest_commit=$(git ls-remote git://github.com/$repository refs/heads/master)
+    latest_commit=$(git ls-remote https://github.com/$repository refs/heads/master)
```

Comment:
Is this change because the `git:` version was failing?  When I use git, the format is instead`git@github.com:$repository` with the `@` before and `:` after `github.com`.  If `https` works then perhaps leave it as you have made it just now, but if there is a reason to stick with the `git` protocol, you could use the `@` and `:` version.

Reply:
The `git@github.com:$repo` is used for pushing later in the script so yes, it would work here. However as far as I can tell there's no reason to favor git over https protocol for fetching.